nautobot/docs/installation/upgrading-from-nautobot-v1.md

```diff
@@ -87,6 +88,8 @@
 
 ### Replaced Models
 
+#### Generic Role Model
```

Comment:
Misleading heading - this section covers both Role and Location changes. Might need to split this into two sections and carve up the "Removed Model/Replaced With" table between those sections if so.